src/components/MapContainer.js

```diff
@@ -80,12 +80,12 @@ class MapContainer extends React.Component {
       .then(coordinates => getTraceAttributes(host, coordinates))
       // This `catch` statement is placed here to handle errors from Fetch API.
       .catch(error => {
+        this.props.stopLoading()
```

Comment:
Oh my bad I removed the stopLoading() at the end of the request. 
Yea, I'm not quite sure which actions would clear all loading actions. I'm thinking I should clear all loading actions wherever in the code there is an error? Would that be correct?